ydb/core/tx/schemeshard/schemeshard__op_traits.h

```diff
@@ -290,4 +299,65 @@ struct TSchemeTxTraits<NKikimrSchemeOp::EOperationType::ESchemeOpCreateBackupCol
     constexpr inline static bool CreateDirsFromName = true;
 };
 
+template <>
+struct TSchemeTxTraits<NKikimrSchemeOp::EOperationType::ESchemeOpBackupBackupCollection> : public TSchemeTxTraitsFallback {
+    static std::optional<THashMap<TString, THashSet<TString>>> GetRequiredPaths(const TTxTransaction& tx, const TOperationContext& context) {
```

Comment:
Very specific and non primitive manipulations here.
I would suggest moving `GetRequiredPaths()` definition closer to the operation itself, to `schemeshard__operation_backup_backup_collection.cpp`.